as2_python_api/as2_python_api/mission_interpreter/mission_stack.py

```diff
@@ -63,6 +58,7 @@ def __str__(self) -> 'MissionItem':
         return current.json() + str(self.pending)
 
     def next_item(self) -> 'MissionItem':
```

Comment:
Did you think about defining __iter__(), __next___() and __len__() method in order to be used as an iterable, e.g. in a for loop ?

Reply:
Sure, I will address them on future pull requests :+1: